### ISSUE TITLE
[BUG] Can't start valheim server because there is no file "valheim_server.x86_64".

### ISSUE THREAD

**Issue by Hogaeng** [GameServerManagers/LinuxGSM#3324]
## User Story

Following tutorial in web page of LGSM, starting has failed because there is not exist valheim_server.x86_64.
Where is that file?
Let me attach script log.
[vhserver-script.log](https://github.com/GameServerManagers/LinuxGSM/files/6070311/vhserver-script.log)

## Basic info

Distro: [Ubuntu 20.04.2 LTS]
Game: Valheim
Command: ./vhserver start
LinuxGSM version: 21.1.3
## Further Information

A clear description of what the bug is and any ideas on how to resolve it. Plus any further context that might be relevant to the issue.

## To Reproduce

Steps to reproduce the behaviour:
1. installed dependencies in admin user, not vhserver user.
2. purged steamcmd and reinstalled it.
3. reinstalled lgsm without removed.
4. add sudo previliage to vhserver user.

**Comment by dgibbs64** [comment]
./vhserver install ?

**Comment by Hogaeng** [comment]
> ./vhserver install ?

I've already done. Please check my script log.

**Comment by Xalorous** [comment]
Can you see the file?
If you run `ls -l ~/serverfiles/valheim_server.x86_64` and paste the results, including the prompt, perhaps we can pinpoint the issue for you.

        [vhserver@sv-valheim ~]$ ls -l serverfiles/valheim_server.x86_64 
        -rwxrwxr-x. 1 vhserver vhserver 6344 Feb 11 20:55 serverfiles/valheim_server.x86_64
        [vhserver@sv-valheim ~]$

**Comment by Hogaeng** [comment]
![image](https://user-images.githubusercontent.com/15939666/109767210-8a968600-7c3a-11eb-97a0-f6c87eacda92.png)
![image](https://user-images.githubusercontent.com/15939666/109767245-984c0b80-7c3a-11eb-85ff-08dd21a65bc9.png)
![image](https://user-images.githubusercontent.com/15939666/109767155-78b4e300-7c3a-11eb-9cf1-6248171e411a.png)
![image](https://user-images.githubusercontent.com/15939666/109766990-44d9bd80-7c3a-11eb-88d3-ce09a5c33ea3.png)

**Comment by Hogaeng** [comment]
![image](https://user-images.githubusercontent.com/15939666/109768079-cd0c9280-7c3b-11eb-93c1-0eba26a65d61.png)
Is it normal downloading folder remains after downloading the game?

**Comment by Svann** [comment]
I have the same error with my fresh install on Ubuntu 20.04.

**Comment by h3o66** [comment]
I have currently no clue why this happens.
Imho something special with steamcmd.
Can you maybe upload the log from the folder `~/.local/share/Steam/logs/` ?

**Comment by Hogaeng** [comment]
I found the reason that I'd installed at GCP f-1 micro which is very low specifications. It's only have 0.6GB so that steamcmd had halted downloading Valheim. I installed at my desktop(amd 2700x, 16GB) using vmware and it works. Also my friend installed and run server in Synology NAS and it works fine too.
I found the minimum specification of running Valheim server in Reddit, check URL below.
I think 2cores and 4GB are good.
[Reddit](https://www.reddit.com/r/valheim/comments/lcg8xk/dedicated_server_requirements/)
